bcdaworker/worker/alr.go

```diff
@@ -200,10 +207,10 @@ func (a *AlrWorker) ProcessAlrJob(
 		return err
 	}
 
-	// If we did not have an ALR data to write, we'll write a specific file name that indicates that the
+	// If we did not have any ALR data to write, we'll write a specific file name that indicates that
```

Comment:
Note that this seems a little too early for the "blank filename" job key to be created, since there is other logic further down in the code that is still run after this job key is created. Wasn't 100% sure if this makes sense to move, decided to keep it as is unless people feel confident about moving this down.

Reply:
yeah, this is a little odd here. Not sure I fully understand why we are continuing job processing if there's no ALR data + the creation of a key here and then more keys later on. If the tests are passing, I am OK to leave this where it is at for now.

When/if this code is reintroduced and/or que-go is replaced, we can do a deeper dive on this?